dashboard/db/migrate/20210908195236_update_backpack_storage_apps_value.rb

```diff
@@ -0,0 +1,10 @@
+class UpdateBackpackStorageAppsValue < ActiveRecord::Migration[5.2]
+  def up
+    PEGASUS_DB[:storage_apps].where(project_type: 'backpack').each do |storage_app|
+      if storage_app[:value] == "".to_json
```

Comment:
Do you want the `to_json` here on the empty string?

Reply:
Yes, it matches the value in the database when I tested it